python/cuml/benchmark/nvtx_benchmark.py

```diff
@@ -188,5 +190,14 @@ def profile(self, command):
 
 
 if __name__ == "__main__":
+    def check_version():
+        stdout = Profiler._execute(['nsys', '--version'])
+        full_version = stdout.decode("utf-8").split(' ')[-1]
+        year, month = full_version.split('.')[:2]
+        version = float(year + '.' + month)
+        if version < 2021.4:
+            raise Exception('The nsys utility needs to be updated')
```

Comment:
Can you add the minimum version needed to the error being thrown? Overall, we need more verbose and helpful errors, so would be good to start even with simple ones :)